tools/generator-go-sdk/internal/generator/helpers.go

```diff
@@ -2,11 +2,31 @@ package generator
 
 import (
 	"fmt"
+	"os"
 	"strings"
 
 	"github.com/hashicorp/pandora/tools/data-api-sdk/v1/models"
 )
 
+func CleanAndRecreateWorkingDirectory(path string) error {
+	// rm -r 💥
```

Comment:
should this be an emoji ?

Reply:
yep, it is a cautionary comment :firecracker: